src/main/java/org/apache/commons/lang3/concurrent/AtomicInitializer.java

```diff
@@ -63,8 +63,10 @@
  * @param <T> the type of the object managed by this initializer class
  */
 public abstract class AtomicInitializer<T> extends AbstractConcurrentInitializer<T, RuntimeException> {
+
+    private final T NO_INIT = (T) new Object(){};
```

Comment:
- Should be `static`, we don't need a new `Object` instance for each new initializer.
- Why is the Object an anonymous class? IOW: `new Object(){};` --> `new Object();`